packages/amplify_authenticator/example/lib/main.dart

```diff
@@ -96,30 +96,37 @@ class _MyAppState extends State<MyApp> {
         ],
       ),
 
-      // Finally, we specify the widget to use once the user is signed in.
-      child: const SignedInScreen(),
-    );
+      // Your MaterialApp should be the child of the Authenticator.
+      child: MaterialApp(
+        title: 'Authenticator Demo',
+        theme: ThemeData.light(),
+        darkTheme: ThemeData.dark(),
+        themeMode: ThemeMode.system,
+        debugShowCheckedModeBanner: false,
+
+        // These lines enable our custom localizations specified in the lib/l10n
+        // directory, which will be used later to customize the values displayed
+        // in the Authenticator component.
+        localizationsDelegates: const [
+          AppLocalizations.delegate,
+        ],
+        supportedLocales: const [
+          Locale('en'), // English
+          Locale('es'), // Spanish
+        ],
 
-    return MaterialApp(
-      title: 'Authenticator Demo',
-      theme: ThemeData.light(),
-      darkTheme: ThemeData.dark(),
-      themeMode: ThemeMode.system,
-      debugShowCheckedModeBanner: false,
-
-      // These lines enable our custom localizations specified in the lib/l10n
-      // directory, which will be used later to customize the values displayed
-      // in the Authenticator component.
-      localizationsDelegates: const [
-        AppLocalizations.delegate,
-      ],
-      supportedLocales: const [
-        Locale('en'), // English
-        Locale('es'), // Spanish
-      ],
-
-      // The Authenticator component should be a descendant of your MaterialApp.
-      home: authenticator,
+        // The Authenticator component must wrap your Navigator component which
+        // can be done using the `builder` method.
+        builder: (context, child) {
+          return AuthenticatorBody(child: child);
```

Comment:
If we have to expose and force the use of AuthenticatorBody like this, maybe we should change the name? Perhaps we could do `AuthenticatorChild` with a positional argument?

Reply:
Updated 👍